content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

```diff
@@ -45,6 +45,12 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
  ``` shell
 $ ghe-dep-graph-enable
 ```
+{% note %}
+
+**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
+
+{% endnote %}
```

Comment:
Thanks for making that change @djdefi.

I can see from [the staging deployment of this page](https://docs-538--djdefi-dep-graph-she.herokuapp.com/en/enterprise-server@2.22/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server) that, because it's within a list item, the note needs to be indented in the markdown to render correctly.

```suggestion
   {% note %}

   **Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."

   {% endnote %}
```